tests/api/test_product.py

```diff
@@ -3961,3 +3961,184 @@ def test_update_or_create_variant_stocks_empty_stocks_data(variant, warehouses):
     stock = variant.stocks.first()
     assert stock.warehouse == warehouses[0]
     assert stock.quantity == 5
+
+
+# Because we use Scalars for Weight this test query tests only a scenario when weight
+# value is passed by a variable
+MUTATION_CREATE_PRODUCT_WITH_WEIGHT_GQL_VARIABLE = """
```

Comment:
Can we remove useless fields from the response in both mutation?

Reply:
Done